hass_nabucasa/remote.py

```diff
@@ -415,11 +415,11 @@ async def connect(self) -> None:
         except asyncio.TimeoutError:
             _LOGGER.error("Timeout connecting to snitun server")
         except SniTunConnectionError as err:
-            can_reconnect = self._snitun and not self._reconnect_task
+            reconnecting = self._reconnect_task is not None
             _LOGGER.log(
-                logging.INFO if can_reconnect else logging.ERROR,
+                logging.ERROR if reconnecting else logging.INFO,
                 "Connection problem to snitun server%s (%s)",
-                ", reconnecting" if can_reconnect else "",
+                ", reconnecting" if not reconnecting else "",
```

Comment:
Should we say that we are reconnecting when we are not reconnecting?

Reply:
We will be reconnecting after this (in the finally clause)

Reply:
Ok.

Reply:
It probably does not add much anyway, remove that line?

Reply:
Not sure. It's only confusing if you're not familiar with this part of the code. The log message as such is ok.

Reply:
On the same note, only add something if you are familiar with the code.
I'll remove it, and keep things simple.